python/cudf/cudf/tests/test_parquet.py

```diff
@@ -214,14 +214,15 @@ def test_parquet_reader_basic(parquet_file, columns, engine):
 @pytest.mark.parametrize("engine", ["cudf"])
 def test_parquet_reader_empty_pandas_dataframe(tmpdir, engine):
     df = pd.DataFrame()
-
     fname = tmpdir.join("test_pq_reader_empty_pandas_dataframe.parquet")
-
     df.to_parquet(fname)
     assert os.path.exists(fname)
-
     expect = pd.read_parquet(fname)
     got = cudf.read_parquet(fname, engine=engine)
+    expect = expect.reset_index(drop=True)
+    got = got.reset_index(drop=True)
```

Comment:
What was the reasoning behind needing these? What type of index does cudf return and what does pandas return?

Reply:
I'll look into this tomorrow and figure out why our Dataframes and pandas are indexed differently.

Reply:
If we don't reset the index an error occurs because cudf returns a float and pandas returns an object. Do you think we change this or should we keep the current fix of resetting the index? 

```
>>> p1 = pd.read_parquet('t')
>>> p2 = cudf.read_parquet('t')
>>> p1.dtypes
Series([], dtype: object)
>>> p2.dtypes
Series([], dtype: float64)
>>> 
```
This is the error I get when I run the test without the index reset
```
l = Index([], dtype='object'), r = Float64Index([], dtype='float64'), obj = 'DataFrame.index'

    def _check_types(l, r, obj="Index"):
        if exact:
>           assert_class_equal(l, r, exact=exact, obj=obj)
E           AssertionError: DataFrame.index are different
E           
E           DataFrame.index classes are not equivalent
E           [left]:  Index([], dtype='object')
E           [right]: Float64Index([], dtype='float64')
```

Reply:
Got it, thanks! This is totally intended since it doesn't make sense from the cuDF side to default to an empty String dtype Index. This should be good to go!